src/tiledb/cloud/vcf/ingestion.py

```diff
@@ -584,7 +558,27 @@ def ingest_samples_udf(
     with Profiler(
         group_uri=dataset_uri, group_member=LOG_ARRAY, id=id, trace=trace
     ) as prof:
-        prof.write("uris", ",".join(sample_uris))
+        prof.write("uris", str(len(sample_uris)), ",".join(sample_uris))
+
+        # Handle missing index
+        def create_index_file_worker(uri: str) -> None:
+            if not find_index(uri):
+                logger.debug("indexing %r", uri)
+                create_index_file(uri)
+
+        with ThreadPool(threads) as pool:
+            pool.map(create_index_file_worker, sample_uris)
+
+        # TODO: Handle un-bgzipped files
+        """
+        new_sample_uris = []
+        for uri in sample_uris:
+            if not is_bgzipped(uri):
+                logger.debug("bgzipping and indexing %r", uri)
+                uri = bgzip_and_index(uri)
+            new_sample_uris.append(uri)
+        sample_uris = new_sample_uris
+        """
```

Comment:
I would either remove this or make it a comment rather than a multiline string.

Reply:
Removing.